src/manifest/mod.rs

```diff
@@ -734,7 +734,7 @@ impl CrateData {
         disable_dts: bool,
         out_dir: &Path,
     ) -> NpmPackage {
-        let data = self.npm_data(scope, false, disable_dts, out_dir);
+        let data = self.npm_data(scope, true, disable_dts, out_dir);
```

Comment:
This should be `false`. The `to_commonjs` method should also use `false`, only `to_esmodules` should be `true`.

Reply:
But then there would be no change in behavior, and the issue as described would not be fixed? Please let me know if there is a better way of fixing this issue!

Reply:
This is the situation in the old code:

* `to_commonjs` -> `true`
* `to_esmodules` -> `false`
* `to_web` -> `false`
* `to_nomodules` -> `false`

This is the situation in your code:

* `to_commonjs` -> `true`
* `to_esmodules` -> `true`
* `to_web` -> `false`
* `to_nomodules` -> `true`

However it is supposed to be like this:

* `to_commonjs` -> `false`
* `to_esmodules` -> `true`
* `to_web` -> `false`
* `to_nomodules` -> `false`

It is only the `bundler` target (`to_esmodules`) which has the extra `_bg.js` file, the rest don't.

Reply:
Thank you for the explanation - I have made the changes. However, I noticed in this and my previous pr that the test `build_with_and_without_wasm_bindgen_debug` keeps failing locally, and now I see that the same has happened on master. Originally I also had this issue on master locally so I thought it was a configuration issue on my side, but as it also fails in the CI I doubt that that is the case. Do you have any idea why this test fails?

Reply:
Yeah, I need to look into why that's failing, you can ignore that.